tests/CMakeLists.txt

```diff
@@ -127,7 +129,7 @@ add_aktualizr_test(NAME commands SOURCES commands_test.cc)
 
 add_aktualizr_test(NAME config SOURCES config_test.cc ARGS ${PROJECT_BINARY_DIR} PROJECT_WORKING_DIRECTORY)
 
-add_aktualizr_test(NAME crypto SOURCES crypto_test.cc PROJECT_WORKING_DIRECTORY)
+add_aktualizr_test(NAME crypto_test SOURCES crypto_test.cc PROJECT_WORKING_DIRECTORY)
```

Comment:
Minor complaint: this means the test will be named "test_crypto_test". A bit ugly, but not worth holding this up for.

Reply:
Hm... Probably changed that fighting with some cmake issues and forgot to revert. Thanks!